lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/interfaces.cpp

```diff
@@ -118,79 +118,8 @@ int16_t ACS711_Current_Bus::read(uint8_t bus, uint8_t mux) { return curr_buffer_
 
 // VLH35_Angles
 
-#define CLOCK_SPEED 100'000u // 100kHz SSI Clock
-
-SPISettings spi_settings(CLOCK_SPEED, MSBFIRST, SPI_MODE1);
-
-volatile uint8_t VLH35_Angle_Bus::curr_id_ = 0;
-volatile uint8_t VLH35_Angle_Bus::spi_buffer_[VLH35_Angle_Bus::BUFFER_SIZE] = {0};
-volatile uint32_t VLH35_Angle_Bus::enc_buffer_[VLH35_Angle_Bus::BUS_SIZE] = {0};
-
-void VLH35_Angle_Bus::init() {
-  SPI.begin();
-  pinMode(clk_p_, OUTPUT);
-  pinMode(sel0_p_, OUTPUT);
-  pinMode(sel1_p_, OUTPUT);
-  pinMode(sel2_p_, OUTPUT);
-  select_enc_(curr_id_);
-  digitalWriteFast(clk_p_,
-                   HIGH); // Set CLK line HIGH (to meet the requirements
-                          // of SSI interface)
-}
-
-void VLH35_Angle_Bus::select_enc_(uint8_t id) {
-  uint8_t sel0 = id & 1;
-  uint8_t sel1 = id >> 1 & 1;
-  uint8_t sel2 = id >> 2 & 1;
-  digitalWrite(sel0_p_, sel0);
-  digitalWrite(sel1_p_, sel1);
-  digitalWrite(sel2_p_, sel2);
-}
-
-void VLH35_Angle_Bus::transfer() {
-  digitalWriteFast(clk_p_,
-                   LOW); // Set CLK line LOW (to inform encoder -> latch data)
```

Comment:
For backward compatability, do not delete the VLH35_Angle_Bus class, rather make a new one for the AMT13 encoders.